src/core/documentation/format.ts

```diff
@@ -0,0 +1,4 @@
+export enum Format {
+    Markdown = "Markdown",
```

Comment:
Other enums in the codebase do not use explicitly assigned string values, I am not sure on the effect this has on serialization, do you know? Is the structurizr API expecting string or int values on the wire?

Reply:
Yeah, odd. Typescript will default to outputting enums as indices unless you explicitly assign a value. I was going off the OpenAPI spec at `https://structurizr.com/help/web-api` which has the field defined as :
```
        format:
          type: string
          description: The content format type.
          enum:
            - Markdown
            - AsciiDoc
```

`InteractionStyle` is defined in a similar way, but apparently integers work just as well. My gut feeling is to follow the spec. Thoughts?

Reply:
I agree with you - following the spec is the better approach here :-) This just means a reminder to myself that I'll have to check other enums as well.